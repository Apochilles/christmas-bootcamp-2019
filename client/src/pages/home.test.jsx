import React from "react";
import { shallow } from "enzyme";
import Home from "./home";

test("renders the home page without errors", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});
