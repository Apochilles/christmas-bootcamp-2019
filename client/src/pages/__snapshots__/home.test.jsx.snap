// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home page without errors 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <header
      className="col-12"
    >
      <img
        alt="logo"
        className="App-logo"
        src="logo.svg"
      />
      <h1>
        Beer List
      </h1>
    </header>
    <div
      className="row"
    >
      <main
        className="col-12"
      >
        <Connect(BeerList) />
      </main>
    </div>
  </div>
</div>
`;
